enviPath_python/enviPath.py

```diff
@@ -15,12 +15,15 @@
 # DEALINGS IN THE SOFTWARE.
 import copy
 from collections.abc import Iterable
+from typing import Union, List
 
 from requests import Session
 from requests.adapters import HTTPAdapter
 from requests.exceptions import *
 
 from enviPath_python.objects import *
+from enviPath_python.objects import SimpleRule, SequentialCompositeRule, ParallelCompositeRule, User, Package, Compound, \
+    Pathway, Reaction, Scenario, Setting, Rule
```

Comment:
Whats the purpose of the duplicated import? Either import only whats needed or all